src/main/java/seedu/address/model/person/Date.java

```diff
@@ -30,20 +30,20 @@
     public final String value;
 
     /**
-     * Constructs an {@code Email}.
+     * Constructs an {@code Date}.
      *
      * @param email A valid email address.
      */
-    public Email(String email) {
+    public Date(String email) {
```

Comment:
The argument name here should probably be changed from `email` to `date`. The JavaDoc should also be changed accordingly.